src/app/(pages)/api/refresh-token/route.ts

```
@@ -0,0 +1,62 @@
+import { createServerSupabaseClient } from "@/lib/supabase/server"
+
+import { createServerSupabaseClient } from "@/lib/supabase/server"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createServerSupabaseClient'. Consider to delete it or rename it.

'createServerSupabaseClient' is defined here:


(lint/suspicious/noRedeclare)